templates/index.md.tmpl

```diff
@@ -23,10 +62,37 @@ By default, the provider will use the Mondoo CLI configuration file to authentic
 configuration file is located at `~/.config/mondoo/mondoo.yml` on Linux and macOS, and `%HomePath%\mondoo\mondoo.yml`
 on Windows.
 
-You can alternatively use the `credentials` field, or any of the following environment variables ordered by precedence.
+You can alternatively, use the following environment variables ordered by precedence:
 
 * `MONDOO_CONFIG_BASE64`
 * `MONDOO_CONFIG_PATH`
 * `MONDOO_API_TOKEN`
 
+If you want to mange the credential as part of your terraform configuration, use the `credentials` field:
+
+```hcl
+provider "mondoo" {
+  credentials = "{json-formatted-credentials}"
+}
+```
+
+## Regions
+
+By default, the provider will use the Mondoo Platform in the US region. To use the EU region, set the `region`
+attribute:
+
+```hcl
+provider "mondoo" {
+  region = "eu"
+}
+```
+
+For dedicated Mondoo Platform installations, set the `endpoint` attribute:
```

Comment:
Do we have an official name for the dedicated instances?

Reply:
"Mondoo Enterprise Cloud" is the right term, I'll adjust